intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -2298,6 +2298,66 @@ def convert_date(value):
     }
 }
 
+# https://www.shadowserver.org/what-we-do/network-reporting/accessible-radmin-report/
+accessible_radmin = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port', convert_int),
+    ],
+    'optional_fields': [
+        ('source.asn', 'asn', convert_int),
+        # ('classification.identifier', 'tag'),  # always 'accessible-radmin' - set in constant_fields
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('source.reverse_dns', 'hostname', validate_to_none),
+        ('protocol.transport', 'protocol'),
+        ('extra.', 'naics', convert_int),
+        ('extra.', 'version', validate_to_none),
+    ],
+    'constant_fields': {
+        'classification.identifier': 'accessible-radmin',
+        'classification.taxonomy': 'vulnerable',
+        'classification.type': 'vulnerable service',
+    }
+}
+
+# https://www.shadowserver.org/what-we-do/network-reporting/caida-ip-spoofer-report/
+# NOTE: The "type" field is included twice with the same values
+caida = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+    ],
+    'optional_fields': [
+        ('source.asn', 'asn', convert_int),
+        # ('classification.identifier', 'tag'),  # always 'ip-spoofer' - set in constant_fields
+        ('classification.identifier', 'infection'),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('source.reverse_dns', 'hostname', validate_to_none),
+        ('extra.', 'type', validate_to_none),
+        ('extra.', 'naics', convert_int),
+        ('extra.', 'sic', convert_int),
+        ('extra.', 'sector', validate_to_none),
+        # FIXME Is is mappable to some classification.* field? Not included in example data.
+        ('extra.', 'family', validate_to_none),
+        ('source.network', 'network', validate_to_none),
+        ('extra.', 'version', validate_to_none),
```

Comment:
That's the IP version which is obvious from the address and network, maybe we can just drop it.

Reply:
It does, but I had to look at the code myself too :)
Using `False` as field name:
https://github.com/certtools/intelmq/blob/6a657269b0e3ac7518d6e7b8e49535ee154e36b9/intelmq/bots/parsers/shadowserver/parser.py#L184-L187
I will add this to the module-docstring in `config.py`.